src/OnnxParser/nodes/indexing.jl

```diff
@@ -15,6 +15,8 @@ onnx_node_to_flux_layer(node::ONNXConcat) = (xs...) -> begin
     cat(xs..., dims=axis)
 end
 
+islinear(node::ONNXConcat) = true
```

Comment:
It looks to me like this makes it so that `net2dense` also adds a linear node for the concat layer?
This seems somewhat redundant, because `node2dense` then adds a concat layer again...